hack/update-generated-agent-proto.sh

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright 2017 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+protoc --proto_path=pkg/agent/api/grpc --go_out=plugins=grpc:pkg/agent/api/grpc pkg/agent/api/grpc/hyperstart.proto pkg/agent/api/grpc/oci.proto
```

Comment:
I believe you also want to add a `-I=$GOPATH/src/github.com/google/protobuf/src/` to this command. Without it, protoc seems to be unable to find the `Empty` message:

```
$ hack/update-generated-agent-proto.sh 
google/protobuf/empty.proto: File not found.
hyperstart.proto: Import "google/protobuf/empty.proto" was not found or had errors.
hyperstart.proto:15:62: "google.protobuf.Empty" is not defined.
hyperstart.proto:16:60: "google.protobuf.Empty" is not defined.
hyperstart.proto:17:54: "google.protobuf.Empty" is not defined.
hyperstart.proto:18:58: "google.protobuf.Empty" is not defined.
hyperstart.proto:25:52: "google.protobuf.Empty" is not defined.
hyperstart.proto:26:56: "google.protobuf.Empty" is not defined.
hyperstart.proto:29:56: "google.protobuf.Empty" is not defined.
hyperstart.proto:30:60: "google.protobuf.Empty" is not defined.
hyperstart.proto:31:62: "google.protobuf.Empty" is not defined.
hyperstart.proto:32:48: "google.protobuf.Empty" is not defined.
hyperstart.proto:33:56: "google.protobuf.Empty" is not defined.
```

Am I missing something ?

Reply:
```
➜  runtimes git:(grpc_proto) protoc --version
libprotoc 3.5.0
➜  runtimes git:(grpc_proto) hack/update-generated-agent-proto.sh
➜  runtimes git:(grpc_proto)
```

Works on my Mac.

Reply:
@sameo Ah I know why you got  the `Empty` errors.

My `protoc` is built from source and installed together with the include files.

If you download the prebuilt `protoc`, and only put the `bin/protoc` binary in your `$PATH` without install the `include/` dir, you will get the import error.

However, I have not generated the code with the getters in the diff yet.

Reply:
@gnawux I installed protoc from my distro protobuf packages (FC27, `protobuf` and `protobuf-compiler`)

Reply:
@sameo Then does it have an `-devel` or `-dev` like package?

And we need figure out why they generate different code.

Reply:
finally I solved the getter issue. I found an older `protoc-gen-go` in a higher priority position in my `$PATH`